test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -23,7 +23,7 @@ namespace Microsoft.ML.Runtime.RunTests
     /// </summary>
     public abstract partial class BaseTestBaseline : BaseTestClass
     {
-        public const decimal Tolerance = 10_000_000;
+        public const int DigitsOfPrecision = 7;
```

Comment:
>public [](start = 8, length = 6)

nit: can it be internal?

Reply:
Maybe, but the existing code already had the previous constant as public.

Reply:
It looks like it is used by inherited classes.

---
In reply to: [222529405](https://github.com/dotnet/machinelearning/pull/1145#discussion_r222529405) [](ancestors = 222529405)